pkg/configschema/schemamap.go

```diff
@@ -1,5 +1,70 @@
 package configschema
 
+import _ "embed"
+
+//go:embed jsonfiles/4_0_0.json
```

Comment:
You can embed the entire folder and create the entire map without listing each version separately - like below
 
https://github.com/aerospike/aerospike-kubernetes-operator/blob/17b6462a23f49c722ed00a9ecade0a6d53581944/controllers/configmap.go#L46